conans/model/conf.py

```diff
@@ -21,6 +21,7 @@
     "tools.cmake.cmaketoolchain:system_version": "Define CMAKE_SYSTEM_VERSION in CMakeToolchain",
     "tools.cmake.cmaketoolchain:system_processor": "Define CMAKE_SYSTEM_PROCESSOR in CMakeToolchain",
     "tools.env.virtualenv:auto_use": "Automatically activate virtualenv file generation",
+    "tools.cmake.cmake_layout.custom_settings": "Name of the settings that will produce a different build folder and different CMake presets names",
```

Comment:
Please, I need a name for the conf, that is not a good one.

Reply:
``tools.cmake.cmake_layout:build_folder_vars`` (could be ``params``, or something like that)

Reply:
and `tools.cmake.cmake_layout:suffix_folder_vars`?

Reply:
I think `tools.cmake.cmake_layout:build_folder_vars` is good enough.

Reply:
I think having the ``build`` in the name is good, as this maps to the ``self.build_folder`` (or ``self.folders.build`` inside the layout)